cmake/OpenCensusHelpers.cmake

```diff
@@ -38,6 +38,18 @@ function(opencensus_test NAME SRC)
   endif()
 endfunction()
 
+# Helper function like bazel's cc_test. Usage:
+#
+# opencensus_benchmark(trace_some_test internal/some_test.cc dep1 dep2...)
+function(opencensus_benchmark NAME SRC)
+  # if(BUILD_TESTING)
+  set(_NAME "opencensus_${NAME}")
+  add_executable(${_NAME} ${SRC})
+  prepend_opencensus(DEPS "${ARGN}")
+  target_link_libraries(${_NAME} "${DEPS}" benchmark)
+  # add_test(NAME ${_NAME} COMMAND ${_NAME}) endif()
```

Comment:
Not sure what to do here, I'd be happy to drop the commented out line, and to not run benchmarks as part of CI.

Reply:
I tried to uncomment that line. I think benchmarks are useful to run during CI.

Reply:
I tried this on my workstation and the benchmarks are responsible for 53 out of 71 seconds of the `ctest` runtime. I would really prefer to not add this delay to CI.

Reply:
ok, removed :)